multianalyzer/file_io.py

```diff
@@ -522,3 +527,13 @@ def save_rebin(filename, beamline="id22", name="id22rebin", topas=None, res=None
                 debug_ds.attrs["interpretation"] = "image"
                 debug_ds.attrs["info"] = "Number of refinement cycle to converge 2theta"
 
+
+def all_entrees(filename):
+    """
```

Comment:
Shouldn't this be `all_entries` ?

Reply:
True. I'll fix it later. Thanks for the merge!